### PR TITLE
Move job control metadata to control/ipppssoot/job.json enabling other control files for  ipppssoot

### PR DESCRIPTION
**Pull request by jaytmiller**
Move job metadata from contrl/ipppssoot to control/ipppssoot/job.json.
Convert comm.control to comm.metadata and relegate comm.control to simple
S3Io on arbitrary prefixes like comm.inputs and comm.outputs.

### CODE REVIEW

File: calcloud/io.py [hunk 1]
```diff
@@ -320,6 +321,16 @@ class InputsIo(S3Io):
     """
 
 
+class ControlIo(S3Io):
+    """ControlIo provides simple standard operations on the processing
+    control store.
+
+    >>> comm = get_io_bundle()
+    >>> comm.metadata.path('lcw303cjq/env') #doctest: +ELLIPSIS
+    's3://.../comtrol/lcw303cjq/env'
```
Comment (bhayden53) [inline calcloud/io.py:330]: typo

Comment (jaytmiller) [reply calcloud/io.py:330]: Hang on,  already fixed  that,  may have forgotten  push

Comment (jaytmiller) [reply calcloud/io.py]: I did forget,  and  pushed  just now.

**Comment by bhayden53** [comment]
lgtm, merge when ready

**Pull request merged**